app/javascript/src/shared/actionCableSubscription.ts

```diff
@@ -83,6 +84,7 @@ export const eventsSubscriber = (appId, cableApp, dispatch, fetchApp) => {
               ?.contentWindow;
             popup && popup.postMessage(data, '*');
           default:
+            dispatch(appEventReceived(data))
```

Comment:
I would prefer to add a specific type of handling for this, **appEvent** is too general, if this event is intended for AppPackages let's name this **AppPackage** event, and handle this  with **app_package** on the data.type